### PR TITLE
Propose changes to User Stories

### PR DESCRIPTION
**Pull request by bryans17**
The purpose of this pull request is to
1. discuss the use of the term "lab tags"
   - I think it might be too implementation/UI specific so I propose to remove those terms.
2. discuss if step 1b2 for both UC3 and UC4 are necessary
   - Would it make more sense to just end the use cases?

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -307,26 +306,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    * 2b1. TAB displays error message (that lab already exists)
 
       Use case ends
 
 **Use case: Filter Students**
 
 **MSS**
 
-1.  TA requests to filter students by a tag 
+1.  TA requests to filter students by a specific lab
```
Comment (geetengtan) [inline docs/DeveloperGuide.md:318]: I'm not sure about this one. The tutor might want to be able to filter students on other tags, not only labs

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -307,26 +306,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
     Use case ends
 
 **Extensions**
 
-* 1a. TA provides non-existent tag
+* 1a. TA provides non-existent lab
   
     * 1a1. TAB shows error message
-    * 1a2. TAB requests for valid tag
+    * 1a2. TAB requests for valid lab
```
Comment (geetengtan) [inline docs/DeveloperGuide.md:328]: Maybe just end the Use Case here?

**Review by geetengtan — COMMENTED** [review]
In my opinion, a lot of the Use Cases can be terminated when an error or warning appears. 
eg.
* 2a. TA provides non-existent lab
    * 2a1. TAB shows error message informing that the lab doesn't exist.
    * ~~2a2. TAB requests for valid lab~~
    * Use Case ends

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (bryans17) [reply docs/DeveloperGuide.md:318]: Referring to our User Guide for v1.2, I think we only planned to have only have a lab as a tag for now. But if we do plan on adding more tag categories in the future.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (bryans17) [reply docs/DeveloperGuide.md:328]: Valid point. I think both ways work here. Which do you think is more natural? @channne

Comment (channne) [reply docs/DeveloperGuide.md:328]: I think we could also modify the use case to include scenarios where the TA stops providing additional input. Maybe we can save these ideas and decide which one is best to implement after we understand the product better?

Comment (bryans17) [reply docs/DeveloperGuide.md:328]: Alright since no one is rejecting it, for now, I guess we can merge this?

Comment (geetengtan) [reply docs/DeveloperGuide.md:328]: agreed

**Review by channne — APPROVED** [review]
LGTM 👍

**Pull request merged**